docs/api-handlers.md

```diff
@@ -0,0 +1,77 @@
+# Api Handlers
+
+Saleor Apps are meant to work in serverless environment, where Cloud Functions are the foundations of server-side code.
+
+Currently, Saleor heavily relies on Next.js, but in the future, other platforms will be supported.
+
+## Required handlers
+
+Saleor requires 2 endpoints to be available for a standalone app:
+
+- Manifest endpoint - Saleor asks for App information like name and permissions
+- Register endpoint - Saleor uses it to exchange authorization data
+
+TODO - any other endpoints?
```

Comment:
@krzysztofwolski can you confirm only manifest and register are required by Saleor?